### ISSUE TITLE
Adding additional data in tooltip

### ISSUE THREAD

**Issue by hishamahamadeynos** [antvis/g2#1236]
* **G2 Version**: 3.2.7
* **Platform**: Windows 10
* **Mini Showcase(like screenshots)**: 
![Screenshot (38)](https://user-images.githubusercontent.com/47521930/54957120-da14a780-4f77-11e9-8347-047c23cfd289.png)

* **CodePen Link**: https://codepen.io/anon/pen/qvwEOb?editors=0010

Hi,
So if you can see in the picture (and also in the CodePen), I want additional data (count) in the tooltip (shown in the brackets). This data is part of the initial data that is being supplied to the graph, but I can't find a way to put it. The way I've used here, by exploiting the formatter, is actually wrong, because it does a direct value comparison, and multiple similar values in the dataset could result in wrong count being shown. 

TL;DR Basically, in the formatter function I want the whole rowdata to be accessible so I can display other data as well.

**Comment by dxq613** [comment]
you can set tooltip field in .tooltip() method of interval like this:

```js
chart.interval().position('x*y').tooltip('c');

chart.interval().position('x*y').tooltip('c*d*e'); // one field one row in tooltip
chart.interval().position('x*y').tooltip('c*d*e',  (c,d,e) => {title: c, name: d, value: e});
```
find more: https://www.yuque.com/antv/g2-docs/api-geom#tooltip
